### DISCUSSION TITLE
Add JSON import function

### DISCUSSION THREAD

**Discussion by KevinGIRAULT** [ckipp01/skan#93] [Feature Request]
Hello,
I've exported a table from Trello and I'm looking for a TUI tool that can import it (it's JSON). It could allow a lot of people to migrate.

**Comment by ckipp01** [comment]
Hey @KevinGIRAULT, thanks for the request! I haven't looked into this at all, but it sounds like a good feature to have.

**Comment by poleguy** [comment]
You could also allow import from nullboard exports. I'd use that right now.
